fitnessllm_dataplatform/entities/queries.py

```
@@ -0,0 +1,16 @@
+
+
+def get_activities(
+    athlete_id: str,
+    data_source: FitnessLLMDataSource,
+    data_stream: FitnessLLMDataStream,
+):
+    return f"""
+        SELECT DISTINCT activity_id
+        FROM dev_metrics.metrics
+        WHERE athlete_id = '{athlete_id}' and data_source = '{data_source.value}' and data_stream = '{data_stream.value}' and status = 'SUCCESS'
+    """
```

Ruff (0.8.2)

Possible SQL injection vector through string-based query construction

(S608)